lib/ViewAllLogsFilters/ViewAllLogsFilters.js

```diff
@@ -85,6 +85,10 @@ export const ViewAllLogsFilters = ({
     label: `${user.firstName} ${user.lastName}`,
   })), 'value');
 
+  const onFilter = (value, dataOptions) => {
+    return dataOptions.filter(option => new RegExp(value, 'i').test(option.label));
+  };
+
```

Comment:
Unit test? It should be dead-simple to write, and you can easily move this function out of the main component body since it's pure.